packages/block-editor/src/components/block-preview/index.js

```diff
@@ -55,9 +55,17 @@ function ScaledBlockPreview( { blocks, viewportWidth } ) {
 
 				// Hack: we need  to reset the scaled elements margins
 				previewElement.style.marginTop = '0';
+
+				__experimentalOnReady( {
+					scale,
+					position: { x: offsetX * scale, y: offsetY },
+					ref: previewRef,
+				} );
 			} else {
 				const containerElementRect = containerElement.getBoundingClientRect();
-				setPreviewScale( containerElementRect.width / viewportWidth );
+				const scale = containerElementRect.width / viewportWidth;
```

Comment:
How come this code got added? Why was it added and what does it do? Doesn't seem directly connected to having a callback.